CHANGELOG.rst

```diff
@@ -2,6 +2,43 @@
 Changelog
 =========
 
+TBD
+---
+
+* New ``ApplyRule`` class created by the ``@handle_urls`` decorator. This is
+  nearly identical with ``OverrideRule`` except:
+
+    * It's now accepting a ``to_return`` parameter which signifies the data
+      container class that the Page Object returns.
+    * Passing a string to ``for_patterns`` would auto-convert it into
+      ``url_matcher.Patterns``.
+    * All arguments are now keyword-only except for ``for_patterns``.
+
+* Modify the call signature of ``handle_urls``:
+
+    * New ``to_return`` parameter which signifies the data container class that
```

Comment:
I think we should downplay it a bit, because most users shouldn't use to_return argument directly. What do you think about documenting it this way? 

1. handle_urls now sets rule.to_return to the item type declared in the ItemPage
2. for advanced use cases, it's possible to change it by using `to_return` argument of `handle_urls` decorator

Reply:
Makes sense. Updated it in https://github.com/scrapinghub/web-poet/pull/84/commits/8efa813ca6cb8a396beafd552d0be01f4ca6a278.